app/views/admin/settings/forms/_authentication.html.erb

```diff
@@ -161,6 +161,7 @@
                              readonly: true %>
           </div>
           <% authentication_available_providers.each do |provider| %>
+            <% next if provider.provider_name == :forem && !FeatureFlag.enabled?(:forem_passport) %>
```

Comment:
I gave this a try locally and it didn't work for me. The problem is that it will skip all providers that aren't `:forem` because of the first half of the condition where `next unless provider.provider_name == :forem`

In order to switch it to an `unless` clause and get rid of the negated `!FeatureFlag.enabled?` it would need to look like this:

```ruby
<% next unless provider.provider_name != :forem || FeatureFlag.enabled?(:forem_passport) %>
```

Meaning that it will skip (`next`) the provider unless it's any of the other ones (`provider.provider_name != :forem`). If it actually is that provider (`:forem`) provider it should still skip unless the feature flag is enabled.

I don't have strong opinions on this one. I'm slightly leaning a bit more towards leaving it as-is but if you (or anyone else) still feel like this other clause is easier to read I'm down to changing it 🙂

Reply:
Ah, right! My mistake, good call 💯  I like it as is as well.